src/main/scala/cognite/spark/v1/SdkV1Relation.scala

```diff
@@ -81,7 +81,7 @@ abstract class SdkV1Relation[A <: Product, I](config: RelationConfig, shortName:
     }
     val updateExternalIds = if (updatesByExternalId.isEmpty) { IO.unit } else {
       val updatesByExternalIdMap = updatesByExternalId
-        .map(u => u.getExternalId.get -> u.into[U].withFieldComputed(_.externalId, _ => None).transform)
+        .map(u => u.getExternalId.get -> u.transformInto[U])
```

Comment:
`withFieldComputed` was silently ignored - `implicit transform` was used, and code emitted no warning that generating a conversion is impossible for abstract types.